packages/admin/src/types/TypeList.vue

```diff
@@ -196,6 +196,15 @@ export default TypeComponent.register([
       return this.isListSource && this.inline &&
         // If there are only compact forms with no labels, don't add spacing
         (!this.isCompact || this.hasLabels)
+    },
+
+    defaultQuery() {
+      const { defaultOrder, defaultOrderDirection } = this.schema
+      return defaultOrder
+        ? {
+          order: `${defaultOrder} ${defaultOrderDirection || 'asc'}`
+        }
```

Comment:
@koskimas I think I'd keep it simpler, and just have people write `defaultOrder: 'name desc'` if they want to control direction. Would you mind?